plugins/woocommerce-blocks/tests/e2e/tests/rating-filter/rating-filter.block_theme.spec.ts

```diff
@@ -0,0 +1,281 @@
+import { cli } from '@woocommerce/e2e-utils';
+
+const blockData = {
+	name: 'Filter by Rating',
+	slug: 'woocommerce/rating-filter',
+	urlSearchParamWhenFilterIsApplied: '?rating_filter=1',
+};
+
+const test = base.extend< {
+	productCollectionPageObject: ProductCollectionPage;
+} >( {
+	productCollectionPageObject: async (
+		{ page, admin, editor, templateApiUtils, editorUtils },
+		use
+	) => {
+		const pageObject = new ProductCollectionPage( {
+			page,
+			admin,
+			editor,
+			templateApiUtils,
+			editorUtils,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( `${ blockData.name } Block`, () => {
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( {
+			name: 'woocommerce/filter-wrapper',
+			attributes: {
+				filterType: 'rating-filter',
+				heading: 'Filter By Rating',
+			},
+		} );
+		await editor.openDocumentSettingsSidebar();
+	} );
+
+	test( "should allow changing the block's title", async ( { page } ) => {
+		const textSelector =
+			'.wp-block-woocommerce-filter-wrapper .wp-block-heading';
+
+		const title = 'New Title';
+
+		await page.fill( textSelector, title );
+
+		await expect( page.locator( textSelector ) ).toHaveText( title );
+	} );
+
+	test( 'should allow changing the display style', async ( {
+		page,
+		editorUtils,
+		editor,
+	} ) => {
+		const stockFilter = await editorUtils.getBlockByName( blockData.slug );
+		await editor.selectBlocks( stockFilter );
+
+		await expect(
+			page.getByRole( 'checkbox', { name: 'Rated 1 out of 5' } )
+		).toBeVisible();
+
+		await page.getByLabel( 'DropDown' ).click();
+
+		await expect(
+			stockFilter.getByRole( 'checkbox', {
+				name: 'In Stock',
+			} )
+		).toBeHidden();
+
+		await expect(
+			page.getByRole( 'checkbox', { name: 'Rated 1 out of 5' } )
+		).toBeHidden();
+
+		await expect( page.getByRole( 'combobox' ) ).toBeVisible();
+	} );
+
+	test( 'should allow toggling the visibility of the filter button', async ( {
+		page,
+		editorUtils,
+		editor,
+	} ) => {
+		const priceFilterControls = await editorUtils.getBlockByName(
+			blockData.slug
+		);
+		await editor.selectBlocks( priceFilterControls );
+
+		await expect(
+			priceFilterControls.getByRole( 'button', {
+				name: 'Apply',
+			} )
+		).toBeHidden();
+
+		await page.getByText( "Show 'Apply filters' button" ).click();
+
+		await expect(
+			priceFilterControls.getByRole( 'button', {
+				name: 'Apply',
+			} )
+		).toBeVisible();
+	} );
+} );
+
+test.describe( `${ blockData.name } Block - with PHP classic template`, () => {
+	test.beforeEach( async ( { admin, page, editor } ) => {
+		await cli(
+			'npm run wp-env run tests-cli -- wp option update wc_blocks_use_blockified_product_grid_block_as_template false'
+		);
+
+		await admin.visitSiteEditor( {
+			postId: 'woocommerce/woocommerce//archive-product',
+			postType: 'wp_template',
+		} );
+
+		await editor.canvas.locator( 'body' ).click();
```

Comment:
This is equivalent to:
```suggestion
		await admin.visitSiteEditor( {
			postId: 'woocommerce/woocommerce//archive-product',
			postType: 'wp_template',
			canvas: 'edit',
		} );
```

Reply:
TIL! Thanks!